code/game/objects/items/id_cards/contractor_ids.dm

```diff
@@ -35,7 +35,7 @@
 			icon_state = "chit_purple"
 		else
 			icon_state = "chit"
-	return 0
+	return 0 //TODO: Why 0, not parent?
```

Comment:
As the comment says, only stumbled upon it doing the refactor, unsure if that was an intentional change if someone knows?